README.md

```diff
@@ -1341,6 +1341,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [shorturl](https://github.com/prologic/shorturl) - Simple URL shortener with very tiny URLs. ([Demo](https://url.mills.io)) `MIT` `Go`
 - [shuri](https://github.com/pips-/shuri) - SHort URI - Lighweight URL shortener. `MIT` `PHP`
 - [url-shortener](https://github.com/cagataycali/url-shortener) `⚠` - Shitty url shortener, emoji and AI powered. `MIT` `Nodejs`
+- [URL-Shortener](https://github.com/sapioit/URL-shortener) - Make your long links short and only uses lowercase, so you can write and spell them faster! ([Demo](http://scurtez.cf)) `GPL-3.0` `PHP`
```

Comment:
Our syntax checks will not expect the final exclamation mark... Please replace it with just a dot.

Sorry for the inconvenience

Reply:
The third commit fixes that. No problem, I know software can be finicky at times.